pycpfcnpj/compatible.py

```diff
@@ -1,10 +1,12 @@
-import sys
 import string
+import six
```

Comment:
I've removed six recently to avoid any external dependencies. It's like a mantra to this tiny lib. That's what I've used sys. @riquellopes

Reply:
i will remove: ;)